tracing/error_reporter_hooks.go

```diff
@@ -20,7 +20,7 @@ type ErrorReporterSpanHook struct {
 }
 
 // OnEnd logs a message and sends err to Sentry if err is non-nil.
-func (h ErrorReporterSpanHook) OnEnd(span *Span, err error) error {
+func (h ErrorReporterSpanHook) OnEnd(span *ServerSpan, err error) error {
```

Comment:
`*ServerSpan` should be `Span` instead?

Reply:
Ya, I just haven't gotten around to it.  All I've _really_ worked on is `span.go`